velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1785,6 +1785,68 @@ TEST_F(StringFunctionsTest, lpad) {
   EXPECT_EQ(invalidPadString + "abc", lpad("abc", 6, invalidPadString));
 }
 
+TEST_F(StringFunctionsTest, hammingDistance) {
+  const auto hammingDistance = [&](std::optional<std::string> left,
+                                   std::optional<std::string> right) {
+    return evaluateOnce<int64_t>("hamming_distance(c0, c1)", left, right);
+  };
+
+  EXPECT_EQ(hammingDistance("", ""), 0);
```

Comment:
Please add tests where the strings are > 12 characters long as they would need out of line allocations also.

Reply:
Sure. Added two tests; one for ascii and one for unicode.